spec/support/model_with_default_values.cr

```diff
@@ -0,0 +1,12 @@
+# columns in migration defined with default values
+# db/migrations/20180802180357_test_defaults.cr
+class ModelWithDefaultValues < BaseModel
+  table :test_defaults do
+    column greeting : String = "Hello there!"
+    column drafted_at : Time = :now
```

Comment:
I'm not 100% sure, but would `Time.utc` work so we don't need a special case for `:now`? The reason I *think* it would work is that we're using the value "as-is" meaning it will call `Time.utc` every time the default is called, and not just when the program is comipled. If that's the case, the `:now` handling can be removed and we can tell people to use `Time.utc`

Reply:
In this case, my thought was that the SchemaEnforcer will eventually check that the value for the column in model and migration are the same. We support using `:now` in the migration. It makes sense in the migration since it just maps to `NOW()`, but my thought is that it would be weird setting it to `NOW()` in the db, but `Time.utc` in the model..... Or maybe that's one of those "overthinking it" moments?

Reply:
Ah I see what you mean. I think that it is probably ok to just check that it is a time for the SchemaEnforcer. IMO not worth the complexity to make it work with a symbol, especially since I'd guess 99% of the time people set the default to now and will use `Time.utc` so will likely be in sync

Reply:
ok, cool. I'll remove the `:now` stuff and then merge after it's all green. Thanks!